src/pages/public_registration/PublicProjectsContent.tsx

```diff
@@ -13,16 +13,29 @@ import {
   getProjectPeriod,
 } from '../registration/description_tab/projects_field/projectHelpers';
 
-const StyledProjectRow = styled.div`
-  background: ${({ theme }) => theme.palette.background.default};
-  padding: 0.5rem 1rem;
-  margin-top: 1rem;
+const StyledProjectGrid = styled.div`
   display: grid;
   grid-template-columns: 2fr auto 1fr auto 1fr auto 1fr;
   column-gap: 1rem;
   align-items: center;
 `;
 
+const StyledHeadingRow = styled(StyledProjectGrid)`
+  padding: 0 1rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.md + 'px'}) {
+    display: none;
```

Comment:
denne er vel litt shady UU-messig ? Ganske sikker på at du må ha med labels selv om det ikke teknisk er en tabell

Reply:
ups, det hadde jeg helt glemt 😬 
skal fikse det for desktop iaff, så se om jeg finner en løsning for mobil også

Reply:
eller, dette er på mobil da, hvor vi ikke har headings. må det likevel være en kobling mot en beskrivelse på hva det er? tenkte først at dette ble noe sånn: 
```
<Typography id="title-heading">Tittel</Typography>
(...)
<Typography labelledBy="title-heading">{project.title}</Typography>
```
Men så har ikke `Typography` noen label-prop, så det er kanskje riktig slik det er likevel? Det er på en måte bare ekstrainformasjon, og ikke noe man egentlig trenger
![bilde](https://user-images.githubusercontent.com/6313468/120650487-21632080-c47e-11eb-81bb-879041063065.png)

Reply:
For tabeller blir det en `<thead><tr><th>xxx</...>`, men det blir heller ikke riktig her siden det ikke er en tabell